feature_apps/src/main/java/cm/aptoide/pt/feature_apps/data/network/model/AppJSON.kt

```diff
@@ -0,0 +1,14 @@
+package cm.aptoide.pt.feature_apps.data.network.model
+
+internal data class AppJSON(
+  var name: String? = null,
```

Comment:
should have the remaining fields

Reply:
I added all primitive fields, when we need the object ones we may add them.